external/urlfrontier/src/main/java/com/digitalpebble/stormcrawler/urlfrontier/StatusUpdaterBolt.java

```diff
@@ -60,6 +62,7 @@
 
 public class StatusUpdaterBolt extends AbstractStatusUpdaterBolt
         implements RemovalListener<String, List<Tuple>>,
+                Runnable,
```

Comment:
Re-using a Storm bolt to be a `Runnable` sounds odd to me :-) personally, I would prefer a inner class or sth like that. Interested to hear @jnioche here.

Reply:
first, thanks @michaeldinzinger for the PR and @rzo1 for the review.
I agree that this feels wrong and given how Storm handles bolts in a multithreaded fashion, could there be side effects?